backend/api/Models/User/AccessRequestAgencyModel.cs

```diff
@@ -1,8 +1,13 @@
 namespace Pims.Api.Models.User
 {
-    public class AccessRequestAgencyModel : CodeModel<int>
+    public class AccessRequestAgencyModel : CodeModel
     {
         #region Properties
+        /// <summary>
+        /// get/set - Primary key to identify agency.
```

Comment:
agency -> access request

Reply:
Actually it is supposed to be the primary key of the agency here.